features/facebookImport/src/components/tabs/tabs.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
```

Comment:
We already have a tab component in polyLook. We might want to keep it mind for future refactoring.

Reply:
I couldn't find it and didn't even use my own in the end

Reply:
Well, we have it here https://github.com/polypoly-eu/polyPod/tree/main/core/utils/poly-look/src/tabs 
OTOH, we do have a mixture of lit-elements and React here and there's not even a mention of introducing poly-look. So fair enough.